Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -213,6 +218,19 @@ def handleNewSignals(self):
         self._updateInfo()
         super().handleNewSignals()
 
+    def set_input_summary(self):
+        n_data = [str(len(self.data[k].table)) for k in self.data.keys()]
+        names = [self.data[k].name for k in self.data.keys()]
+        tables = [self.data[k].table for k in self.data.keys()]
+
+        summary, details, kwargs = self.info.NoInput, "", {}
+        if self.data:
+            data_list = [(name, table) for name, table in zip(names, tables)]
+            summary = ", ".join(n_data)
+            details = format_multiple_summaries(data_list)
```

Comment:
This can be simplified by passing `zip(names, tables)` instead of `data_list`.
If you insist on passing a list: `list(zip(names, tables))`.